src/structure/shapes.ts

```diff
@@ -15,6 +15,7 @@ import assert from 'assert';
 
 import { default as $3Dmol } from '3dmol';
 import { Quaternion } from '3dmol';
+type ColorSpec = number | string | { r: number; g: number; b: number; a?: number }; // mysterious compile errors if I import from 3dmol
```

Comment:
```suggestion
import { ColorSpec, Quaternion } from '3dmol';
```

this worked fine for me, are you still having compilation errors?

Reply:
yes still getting compilation errors